Procfile

```diff
@@ -0,0 +1 @@
+release: bundle exec rails db:migrate db:seed
```

Comment:
It also requires `db:seed:solidus_bolt`, doesn't it?

Reply:
It didn't work. I had to fix it with this [PR](https://github.com/nebulab/bolt_demo/pull/20).

Now it works as expected https://solidus-bolt-demo.herokuapp.com/admin/bolt/edit .